A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf

```diff
@@ -318,6 +319,8 @@ if (!_busy) then {
                     + (_faction get "vehiclesPlanesCAS")
                     + (_faction get "vehiclesPlanesAA")
                     + (_faction get "vehiclesPlanesTransport");
+		    		+ (_faction get "vehiclesPlanesGunship");
```

Comment:
This should probably be getOrDefault, incase the current template has no vehiclesPlanesGunship